### PR TITLE
Monitoring

### PR DESCRIPTION
**Pull request by IlnurHA**

### CODE REVIEW

File: random_generator/settings.py [hunk 1]
```diff
@@ -1,91 +1,100 @@
...
+    # Current environment
+    environment: str = "dev"
+
+    log_level: LogLevel = LogLevel.INFO
+    # Variables for the database
+    db_file: Path = TEMP_DIR / "db.sqlite3"
+    db_echo: bool = False
+
+    # Variables for Redis
+    redis_host: str = "random_generator-redis"
```
Comment (zaqbez39me) [inline random_generator/settings.py]: Not used now. Could be removed

File: random_generator/settings.py [hunk 2]
```diff
@@ -1,91 +1,100 @@
...
+
+        :return: database URL.
+        """
+        return URL.build(
+            scheme="sqlite+aiosqlite",
+            path=f"///{self.db_file}",
+        )
+
+    @property
+    def redis_url(self) -> URL:
```
Comment (zaqbez39me) [inline random_generator/settings.py]: And this could be removed

File: random_generator_front_end/streamlit_app.py [hunk 3]
```diff
@@ -20,10 +20,15 @@ def fetch_from_backend(client: Client, selected_source: str) -> None:
     result = client.get(f"/api/random/{selected_source}")
     if result.status_code == 200:
         result_json: dict = ujson.loads(result.text)
-        if not isinstance(result_json, dict) or "number" not in result_json.keys():
+
+        if (
```
Comment (zaqbez39me) [inline random_generator_front_end/streamlit_app.py:24]: Maybe, better to use pydantic here. But it is ok

**Comment by IlnurHA** [comment]
@zaqbez39me Removed unnecessary env variables from settings. Please, check

**Review by zaqbez39me — APPROVED** [review]

**Comment by zaqbez39me** [comment]
@IlnurHA thanks

**Pull request merged**